<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogHistoricalDataTimeSet</class>
 <widget class="QDialog" name="DialogHistoricalDataTimeSet">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>345</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="pushButton_EndTime">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>23:59</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>起始时间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>表格日期</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Date">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>结束时间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_StartTime">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SamplingPeriod">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>30</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>采样周期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>毫秒</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
